### ISSUE TITLE
Possible failed PSU in angor

### ISSUE THREAD

**Issue by tomhughes** [openstreetmap/operations#994]
It looks like a PSU in angor failed around 09:50 UTC today - either that or it has lost power which did happen for a few days back in March.

**Comment by Firefishy** [comment]
Yes, there has been an ongoing problem with power on machine. I am uncertain if PSU fault or Cable/Connection issue or Data Center power issue.

**Comment by Firefishy** [comment]
PSU part number is: 0Y8Y65

**Comment by Firefishy** [comment]
Cold reboot of the BMC has fixed the issue. The same issue happened back in March 2023, site inspection reported both PSUs reporting OK.

```
sudo ipmitool sdr type "Power Supply"
PS Redundancy    | 74h | ok  |  7.1 | Fully Redundant
Status           | 62h | ok  | 10.1 | Presence detected
Status           | 63h | ok  | 10.2 | Presence detected
```
